falcon/app.py

```diff
@@ -505,6 +505,10 @@ def add_route(self, uri_template, resource, **kwargs):
                 corresponding request handlers, and Falcon will do the right
                 thing.
 
+                Note:
+                    When using an async version of the ``App``, all request
```

Comment:
This `an async` -> `the async` sounds better to me

Reply:
It could be read both ways; I've copied this fragment from another docstring in the `App` class.
I'm open to changing it either way.

Reply:
I'm sure it's fine this way then